privaterelay/templates/includes/login.html

```diff
@@ -13,20 +13,20 @@
 	 data-ga="send-ga-funnel-pings"
 	 data-event-category="Sign In Buttons"
 	 data-event-label="sign-in-btn-header"
-	 class="sign-up button"
+	 class="sign-up button header-link"
 	 href="{% provider_login_url 'fxa' process='login' %}">{% ftlmsg 'nav-profile-sign-up' %}</a>
     <a
       data-entrypoint="relay-sign-in-header"
       data-form_type="button"
       data-ga="send-ga-funnel-pings"
       data-event-category="Sign In Buttons"
       data-event-label="sign-in-btn-header"
-      class="mzp-c-button mzp-t-secondary mzp-t-dark home-sign-in"
+      class="mzp-c-button mzp-t-secondary mzp-t-dark u-font-metropolis u-font-weight-normal"
```

Comment:
Following Protocol naming convention here: https://github.com/mozilla/fx-private-relay/blob/main/docs/coding-standards.md#class-naming-convention

Opted for `u-` utility classes since I'm overwriting certain Protocol button styling attributes.